cache/README.md

```diff
@@ -0,0 +1,59 @@
+# Cache
+
+## **Requests**
+
+|          Route           |                       Description                       |
+| :----------------------: | :-----------------------------------------------------: |
+| [GET /cache](#get-cache) | Returns the cloudflare cache meta data of last 24 hours |
+
+---
+
+## **GET /cache**
+
+Returns the cache meta data and its count of last 24 hours.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  rds-session: `<JWT>`
+
+- **Success Response:**
+
+  - **Code:** 200
+
+    - **Content:**
+
+    ```json
+    {
+      "clearedCount": "number",
+      "lastCleared": "<ISO 8601 timestamp>",
+      "timeFrom": "<ISO 8601 timestamp>",
+      "timeTo": "<ISO 8601 timestamp of 'NOW'>"

```

Comment:
What is the use of `timeFrom` `timeTo`?

Reply:
timeFrom and timeTo is the applicable duration of the last slot like from when to when the last cleared slot will be freezed.

Reply:
Or we can simply say timeTo is when the next slot will be available